packages/obojobo-document-engine/src/scripts/oboeditor/documents/oboeditor-tutorial.json

```diff
@@ -7918,32 +8034,46 @@
 							"content": {
 								"textGroup": [
 									{
-										"data": {
-											"align": "left"
-										},
+										"data": {},
 										"text": {
-											"value": "The review type describes how the student will see their answers after the attempt is over.  Selecting never ensures that, after every attempt, the student will only see which questions they got wrong, but will not be able to see the correct answers.  Selecting always allows the student to see what they got wrong and what the correct answer was after every attempt.  A value of no-attempts-remaining allows the student to see which question they got wrong after every attempt, but the review does not show them the correct answers until after they have exhausted all of their attempts.",
+											"value": "The review type describes how answers will be displayed after an attempt has ended.",
+											"styleList": []
+										}
+									},
+									{
+										"data": {},
+										"text": {
+											"value": "A value of never means that the student will only see whether or not they got a question wrong after every attempt. Correct answers will not be shown.",
 											"styleList": [
 												{
-													"end": 108,
-													"data": {},
-													"type": "monospace",
-													"start": 103
-												},
-												{
-													"end": 268,
+													"end": 16,
 													"data": {},
 													"type": "monospace",
-													"start": 262
-												},
+													"start": 11
+												}
+											]
+										}
+									},
+									{
+										"data": {},
+										"text": {
+											"value": "A value of always means that the student will see both incorrect responses and the correct answers after every attempt.",
```

Comment:
A value of always means that for every attempt, the student will always see the correct answers.